### PR TITLE
Update all deprecated `aes_string()` call to `aes()` with .data pronoun

### PR DESCRIPTION
**Pull request by Felixmil**
Followed ggplot2 recommendation for inclusion in this package. 
As stated in #423 `aes_string()` is deprecated and using `aes()` will allow us to do more things in a simpler way.

> **Note**
> All tests passed on `{osps}` after installing this branch as local version. However, since this change impact basically all the plotting function, it may be considered as a breaking change.

### CODE REVIEW

File: R/plot-histogram.R
Comment (svavil) [inline R/plot-histogram.R:136]: Is `coun` a mistype here?

**Comment by svavil** [comment]
Is it expected that many plot markers change from squares to circles? Why is that?

**Comment by Felixmil** [comment]
> Is it expected that many plot markers change from squares to circles? Why is that?

This is comming from #414, please pull latest version of the branch.

**Comment by svavil** [comment]
> > Is it expected that many plot markers change from squares to circles? Why is that?
> 
> This is comming from #414, please pull latest version of the branch.

I don't mean that the markers change in my local testing, but rather than #424 includes these changes in the test png files.

**Comment by Felixmil** [comment]
> > > Is it expected that many plot markers change from squares to circles? Why is that?
> > 
> > 
> > This is comming from #414, please pull latest version of the branch.
> 
> I don't mean that the markers change in my local testing, but rather than #424 includes these changes in the test png files.

- #414  was merged into develop while #418 was waiting for merge. #414 changed shape level 1 from square to points which made the #414 test png outdated,
- #418 was merged without generating new test png,
- `develop`, containing outdated pngs was merged into this branch,
- Test run in this branch updated the pngs which were committed in order to make the build pass on appveyor.

**Comment by PavelBal** [comment]
Fine with me, but I think @pchelle should review it first.

**Comment by PavelBal** [comment]
@pchelle Please review at your earliest convenience :)

File: R/plot-histogram.R
Comment (Felixmil) [reply R/plot-histogram.R]: I think using {lintr} would be great. There's already a Github action ready to go [here](https://github.com/r-lib/actions/tree/old/examples#lint-workflow).

We would need to customize it a little and define which [linter](https://lintr.r-lib.org/reference/linters.html) we want to apply. But that depends on the scope of the verification we want to implement.

Maybe we can start with the most basic level: make sure that the code can be evaluated. I can make some test if required.

**Review by PavelBal — APPROVED** [review]

**Comment by PavelBal** [comment]
@Yuri05 I am going to merge today/tomorrow, if no one objects.

**Comment by Yuri05** [comment]
@PavelBal I would like to have the approval by @pchelle before merging

**Comment by PavelBal** [comment]
> I would like to have the approval by @pchelle before merging

Sure, let's wait then.

**Comment by msevestre** [comment]
@pchelle Please review asap. Thanks

**Review by pchelle — APPROVED** [review]
All good.
`after_stat` was introduced in ggplot2 starting version 3.3.0, so there is no compatibility issue.
should we declare `ggplot2::after_stat` instead to stay consistent with all the other calls to ggplot2 ?

**Comment by Felixmil** [comment]
> should we declare `ggplot2::after_stat` instead to stay consistent with all the other calls to ggplot2 ?

Yes. Done in latest commit.

**Pull request merged**
